<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>报名工具助手</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>760</width>
        <height>510</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>160</width>
         <height>512</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>512</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 243, 248);
</string>
       </property>
       <widget class="QLabel" name="version">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>420</y>
          <width>91</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(122, 131, 150);
border-radius:5px;</string>
        </property>
        <property name="text">
         <string>版本号 V1.1.2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>440</y>
          <width>101</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border:none;
	color: rgb(0, 112, 249);
	font: 700 12pt &quot;Microsoft YaHei UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>报名工具</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>130</y>
          <width>130</width>
          <height>221</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="plusSet_2">
           <property name="minimumSize">
            <size>
             <width>128</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(142, 171, 221);
	border-radius:5px;
	color: rgb(255, 255, 255);
font: 12pt &quot;Microsoft YaHei UI&quot;;
border-radius: 10px;
}
QPushButton:hover
{
	background-color: rgb(224, 230, 238);
	border-radius:5px;
font: 12pt &quot;Microsoft YaHei UI&quot;;
}</string>
           </property>
           <property name="text">
            <string>软件设置</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RemoteControl">
           <property name="minimumSize">
            <size>
             <width>128</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(142, 171, 221);
	border-radius:5px;
	color: rgb(255, 255, 255);
font: 12pt &quot;Microsoft YaHei UI&quot;;
border-radius: 10px;
}
QPushButton:hover
{
	background-color: rgb(224, 230, 238);
	border-radius:5px;
font: 12pt &quot;Microsoft YaHei UI&quot;;
}</string>
           </property>
           <property name="text">
            <string>扫码登录</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="EquipmentList">
           <property name="minimumSize">
            <size>
             <width>128</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(142, 171, 221);
	border-radius:5px;
	color: rgb(255, 255, 255);
font: 12pt &quot;Microsoft YaHei UI&quot;;
border-radius: 10px;
}
QPushButton:hover
{
	background-color: rgb(224, 230, 238);
	border-radius:5px;
font: 12pt &quot;Microsoft YaHei UI&quot;;
}</string>
           </property>
           <property name="text">
            <string>选择讲座</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="plusSet">
           <property name="minimumSize">
            <size>
             <width>128</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(142, 171, 221);
	border-radius:5px;
	color: rgb(255, 255, 255);
font: 12pt &quot;Microsoft YaHei UI&quot;;
border-radius: 10px;
}
QPushButton:hover
{
	background-color: rgb(224, 230, 238);
	border-radius:5px;
font: 12pt &quot;Microsoft YaHei UI&quot;;
}</string>
           </property>
           <property name="text">
            <string>报名讲座</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>0</y>
         <width>601</width>
         <height>511</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>601</width>
         <height>511</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="login" name="loginPage"/>
       <widget class="useSettings" name="useSettingsPage"/>
       <widget class="chooseALecture" name="chooseALecturePage"/>
       <widget class="QWidget" name="registerLecturesPage"/>
       <widget class="waitPost" name="waitPostPage"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>login</class>
   <extends>QWidget</extends>
   <header>login.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>chooseALecture</class>
   <extends>QWidget</extends>
   <header>choosealecture.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>waitPost</class>
   <extends>QWidget</extends>
   <header>waitpost.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>useSettings</class>
   <extends>QWidget</extends>
   <header>usesettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
